src/go/k8s/apis/redpanda/v1alpha1/redpanda_clusterspec_types.go

```diff
@@ -207,47 +257,52 @@ type Requests struct {
-	CloudStorageUploadCtrlPCoeff            int    `json:"cloud_storage_upload_ctrl_p_coeff,omitempty"`
-	CloudStorageUploadCtrlUpdateIntervalMs  int    `json:"cloud_storage_upload_ctrl_update_interval_ms,omitempty"`
+	CloudStorageEnabled                     *string `json:"cloud_storage_enabled,omitempty"`
+	CloudStorageAPIEndpoint                 *string `json:"cloud_storage_api_endpoint,omitempty"`
+	CloudStorageAPIEndpointPort             *int    `json:"cloud_storage_api_endpoint_port,omitempty"`
+	CloudStorageBucket                      *string `json:"cloud_storage_bucket"`
+	CloudStorageCacheCheckInterval          *int    `json:"cloud_storage_cache_check_interval,omitempty"`
+	CloudStorageCacheDirectory              *string `json:"cloud_storage_cache_directory,omitempty"`
+	CloudStorageCacheSize                   *string `json:"cloud_storage_cache_size,omitempty"`
+	CloudStorageCredentialsSource           *string `json:"cloud_storage_credentials_source,omitempty"`
+	CloudStorageDisableTLS                  *bool   `json:"cloud_storage_disable_tls,omitempty"`
+	CloudStorageEnableRemoteRead            *bool   `json:"cloud_storage_enable_remote_read,omitempty"`
+	CloudStorageEnableRemoteWrite           *bool   `json:"cloud_storage_enable_remote_write,omitempty"`
+	CloudStorageInitialBackoffMs            *int    `json:"cloud_storage_initial_backoff_ms,omitempty"`
+	CloudStorageManifestUploadTimeoutMs     *int    `json:"cloud_storage_manifest_upload_timeout_ms,omitempty"`
+	CloudStorageMaxConnectionIdleTimeMs     *int    `json:"cloud_storage_max_connection_idle_time_ms,omitempty"`
+	CloudStorageMaxConnections              *int    `json:"cloud_storage_max_connections,omitempty"`
+	CloudStorageReconciliationIntervalMs    *int    `json:"cloud_storage_reconciliation_interval_ms,omitempty"`
+	CloudStorageRegion                      *string `json:"cloud_storage_region"`
+	CloudStorageSegmentMaxUploadIntervalSec *int    `json:"cloud_storage_segment_max_upload_interval_sec,omitempty"`
+	CloudStorageSegmentUploadTimeoutMs      *int    `json:"cloud_storage_segment_upload_timeout_ms,omitempty"`
+	CloudStorageTrustFile                   *string `json:"cloud_storage_trust_file,omitempty"`
+	CloudStorageUploadCtrlDCoeff            *int    `json:"cloud_storage_upload_ctrl_d_coeff,omitempty"`
+	CloudStorageUploadCtrlMaxShares         *int    `json:"cloud_storage_upload_ctrl_max_shares,omitempty"`
+	CloudStorageUploadCtrlMinShares         *int    `json:"cloud_storage_upload_ctrl_min_shares,omitempty"`
+	CloudStorageUploadCtrlPCoeff            *int    `json:"cloud_storage_upload_ctrl_p_coeff,omitempty"`
+	CloudStorageUploadCtrlUpdateIntervalMs  *int    `json:"cloud_storage_upload_ctrl_update_interval_ms,omitempty"`
 }
 
 // TieredStoragePersistentVolume is a top level field of the values file
 type TieredStoragePersistentVolume struct {
 	Annotations  map[string]string `json:"annotations,omitempty"`
-	Enabled      bool              `json:"enabled"`
```

Comment:
is this now removed?

Reply:
Yes, Please take a look https://github.com/redpanda-data/helm-charts/pull/756/files#diff-df34075ea48d6120c348021a3727de5f513afd338dfa2e8378993b5b1f374d39R407